apps/question-the-docs/backend/ai/utils/github.py

```diff
@@ -53,41 +73,20 @@ def download_and_decode(url):
     return base64.b64decode(blob['content'])
 
 
-def save_github_md_files_locally(repo_details):
-    print(f"Downloading files from GitHub for {json.dumps(repo_details)}")
-    owner = repo_details['owner']
-    name = repo_details['repo']
-    branch = repo_details['branch']
-    documentation_location = repo_details['documentation_location']
+def save_github_md_files_locally(repo):
+    owner, name, branch, documentation_location = REPOS[repo].values()
 
     repo_contents = gh_repo_contents(owner, name, branch)
     urls = documentation_markdown_urls(repo_contents, documentation_location)
 
     try:
         Path(f"docs/{name}").mkdir(exist_ok=False, parents=True)
```

Comment:
If you don't care if it exists, then don't do the try/catch, just set `exist_ok=True`.